README.md

```diff
@@ -42,5 +45,7 @@ Then run:
 ```
 bundle
 bundle exec rake db:create
+bundle exec rake db:migrate
+bundle exec rake slack:update
```

Comment:
@arun057 isn't that going to happen only once? if it is then we can skip cron for now